extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/sql/resources/SqlStatementResource.java

```diff
@@ -536,8 +536,8 @@ private Optional<ResultSetInformation> getSampleResults(
         rows = 0L;
         size = 0L;
         for (PageInformation pageInformation : pageList.get()) {
-          rows += pageInformation.getNumRows();
-          size += pageInformation.getSizeInBytes();
+          rows += pageInformation.getNumRows() != null ? pageInformation.getNumRows() : 0L;
```

Comment:
nit:

From the code, it seems like numRows can't be null, so can we modify the `PageInformation` instead to remove the `Nullable` tag for `numRows`, convert it to a long, and remove the check here?

Similarly, with this change, we aren't using the nullity of `PageInformation::sizeOfBytes()` anywhere apart from this check, therefore can we convert it to a primitive long, populate it as 0 instead of `null`, remove the `Nullable` and clean up the special handling here?
 
Since we don't require any special ability of the boxed types, it seems better to clean them up. WDYT?

Reply:
Since page information is serialized as json in the `SqlStatementResult`, I prefer `PageInformation#numRows` and `sizeOfBytes` to be Long. That way if they are not populated for whatever reason, they are not sent across to the end users.

Reply:
In that case, please add `@JsonInclude(JsonInclude.Include.NON_NULL)` to the Jackson annotations of the members of `PageInformation`. By default, we do serialize and present null values to the user. So per the current behavior, page information would get serialized as `{"numRows":null,"sizeInBytes":null,"id":5}`. 

Approving the PR since the fix is not related to the current change.

Reply:
Good catch. Will do that in the next PR.